docs/enterprise-guide/data-sandboxes.md

```diff
@@ -126,27 +126,34 @@ Now, when I log in as this user and look at the `Orders` table, I only see the c
 
 ![Results](images/sandboxing/advanced-example-2-results.png)
 
-### Current limitations
+## Current limitations
 
-#### A user can only have one sandbox per table
+### A user can only have one sandbox per table
 
 I.e., if a user belongs to two user groups, both of which have been given sandboxed access to the same table, that user will not be able to access data from that table. You will either need to remove that user from one of those groups, or remove the sandboxed access from one of those groups.
 
-#### Data sandboxes do not support non-SQL databases
+### Data sandboxes do not support non-SQL databases
 
 You can't currently create data sandboxes with non-SQL databases such as Google Analytics, Apache Druid, or MongoDB .
 
-#### Data sandboxes do not work on SQL/native queries
+### Data sandboxes don't work for groups that can execute SQL/native queries
 
-This goes for both saved or unsaved SQL queries. The reason is that Metabase does not currently parse the contents of SQL queries, and therefore can't know conclusively which table(s) are referenced in a query. There are three specific situations to be aware of when it comes to SQL and sandboxes:
+If you're using data sandboxing, your groups should _not_ have access to:
```

Comment:
I would say something like: "the groups with the users that you want sandboxed", to be a bit more specific